runtime/flags/flags.json

```diff
@@ -0,0 +1,145 @@
+[
+  {
+    "name": "broadcast-channel",
+    "text": "Enable unstable `BroadcastChannel` API",
+    "showInHelp": true,
+    "kind": "runtime"
```

Comment:
Consider splitting flags into `runtime: {}` and `cli: {}` subdicts instead of `kind` field.